docker/GPU/Dockerfile

```diff
@@ -66,9 +66,11 @@ RUN git clone https://github.com/open-mmlab/mmdeploy &&\
     pip install -e .
 
 ### build sdk
-RUN git clone https://github.com/openppl-public/ppl.cv.git &&\
+RUN wget https://github.com/openppl-public/ppl.cv/archive/refs/tags/v0.6.1.zip &&\
+    unzip v0.6.1.zip && mv ppl.cv-0.6.1 ppl.cv &&\
```

Comment:
Hi, could the tag `0.6.1` be an `ARG` value at the top of Dockerfile?

Reply:
ok will make the change

Reply:
> could update the doc as well https://github.com/open-mmlab/mmdeploy/blob/master/docs/en/build.md#dependencies

made both the changes